src/controllers/midi/midicontroller.cpp

```diff
@@ -193,6 +198,15 @@ QString formatMidiMessage(unsigned char status, unsigned char control, unsigned
                          QString::number((status & 255)>>4, 16).toUpper(),
                          QString::number(control, 16).toUpper().rightJustified(2,'0'),
                          QString::number(value, 16).toUpper().rightJustified(2,'0'));
+        case MIDI_START:
+            return QString("MIDI status 0x%1: Start Sequence")
+                    .arg(QString::number(status, 16).toUpper());
+        case MIDI_TIMING_CLK:
```

Comment:
IMHO we should unify the wording towards the Midi standard and replace all "pulse" and "tick" with just   
"clock"
https://en.wikipedia.org/wiki/MIDI_beat_clock